src/ucp/core/ucp_ep.c

```diff
@@ -2457,16 +2457,18 @@ ucp_ep_is_am_keepalive(ucp_ep_h ucp_ep, ucp_rsc_index_t rsc_idx)
            (rsc_idx != UCP_NULL_RESOURCE) && /* rsc index is not NULL, i.e.
                                                 it is not a CM lane */
            (ucp_worker_iface(ucp_ep->worker, rsc_idx)->attr.cap.flags &
-            UCT_IFACE_FLAG_CONNECT_TO_IFACE); /* connect-to-iface */
+            UCT_IFACE_FLAG_CONNECT_TO_IFACE) && /* connect-to-iface */
+           (ucp_worker_iface(ucp_ep->worker, rsc_idx)->attr.cap.flags &
```

Comment:
resolve ucp_worker_iface(ucp_ep->worker, rsc_idx)->attr once and save to *iface_attr

Reply:
fixed